src/Microsoft.DotNet.Tools.New/Program.cs

```diff
@@ -24,13 +30,27 @@ private static string GetFileNameFromResourceName(string s)
             return parts[parts.Length - 2] + "." + parts[parts.Length - 1];
         }
 
-        public int CreateEmptyProject()
+        public int CreateEmptyProject(Language language)
```

Comment:
I prefer to make types than string interpolation

There embedded files are used, no problems about directory traversal ( all parts are dot separated as a single name )

I'll whitelist strings ( c# and f# ) for safety

It's ok use string directly, we are using these only to get the directory name of resources, so less translation string -> enum -> string